pipelines/google_ads/__init__.py

```diff
@@ -0,0 +1,141 @@
+from typing import Dict, List, Union
+import dlt
+from dlt.common.configuration.specs import GcpClientCredentialsWithDefault
+from pipelines.google_analytics.helpers.credentials import GoogleAnalyticsCredentialsOAuth
```

Comment:
please do not import cross-pipelines. we have good support for OAUTH now. Import `GcpOAuthCredentialsWithDefault` from specs. it is similar to yours

Reply:
done